info-compiler/src/main/java/com/google/sps/infocompiler/Config.java

```diff
@@ -21,6 +21,6 @@
   public static final String CIVIC_INFO_API_KEY = "";
   public static final String CUSTOM_SEARCH_KEY = "";
   public static final String CUSTOM_SEARCH_ENGINE_ID = "";
-  public static final String OpenNLP_SENTENCE_DETECTOR_FILE = "en-sent.bin";
-  public static final String OpenNLP_TOKENIZER_FILE = "en-token.bin";
+  public static final String OpenNLP_SENTENCE_DETECTOR_FILE = "/home/feiyangyu/en-sent.bin";
+  public static final String OpenNLP_TOKENIZER_FILE = "/home/feiyangyu/en-token.bin";
```

Comment:
Will this be true for the committed code?

Reply:
Sorry for the mishap! This got changed in the branch chained to this one: https://github.com/googleinterns/voter-central/blob/kathy_InfoCompiler_home_address/info-compiler/src/main/java/com/google/sps/infocompiler/Config.java

Reply:
I'll make the change here as well